bin/stencil-start

```diff
@@ -230,7 +243,6 @@ function startServer() {
             },
             proxy: "localhost:" + stencilServerPort,
             tunnel: tunnel,
-            https: Url.parse(staplerUrl).protocol === 'https:'
```

Comment:
Is there a reason that we are removing https?

Reply:
This causes issues with stencil start since it forces the users to use https and http is unavailable. Also it causes issue with browser sync UI to not load